install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2669,7 +2669,7 @@ auth:
           # -- SPIRE agent labels
           labels: { }
           # -- SPIRE Workload Attestor kubelet verification.
-          skipKubeletVerification: false
+          skipKubeletVerification: true
```

Comment:
This does worry me a little bit, in that we're already doing some funky stuff with attestation in having the Cilium agent have heaps of the process delegated to it.

Reply:
That said, now I wrote that, maybe it is less of a problem because the kubelet is not actually involved in any attestation aside from the Cilium Agent and Operator getting their identities.

Reply:
My original approach is to go with secured sensible default values, and user can still change it via helm values based on k8s distro they are using (e.g. kind, minikube).